liota/dccs/dcc.py

```diff
@@ -69,9 +68,9 @@ def _create_relationship(self, entity_parent, entity_child):
     def _format_data(self, reg_metric):
         pass
 
+    @abstractmethod
     def publish(self, reg_metric):
-        message = self._format_data(reg_metric)
```

Comment:
Publish should be an abstract method as it has to be overidden by the base class and implementation goes in the base class.